scripts/local-universe.py

```diff
@@ -27,18 +27,6 @@ def main():
         print("You must run this as root, please `sudo` first.")
         sys.exit(1)
 
-    # jsonschema is required by the universe build process, make sure it is
-    # installed before running.
-    if not shutil.which("jsonschema"):
-        print("You must first install jsonschema (pip install jsonschema).")
-        sys.exit(1)
-
-    # cosmos requires directories to be saved. python does it only sometimes.
-    # Use zip to make sure it works.
-    if not shutil.which("zip"):
-        print("You must first install `zip`.")
-        sys.exit(1)
-
```

Comment:
Don't need to check for jsonschema because we now install it.

The program `zip` is not used. We rely on `gen-universe.py`. Must be a legacy check.